CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## x.x.x
+* Fix `adViewId` not correctly set for banners and MRECs (`<AdView/>`) on Android.
```

Comment:
What are symptoms?

Reply:
It actualy works.  I guess it is because of the annotation.  But, this needs to be fixed anyway.